docs/spelling_wordlist.txt

```diff
@@ -13,6 +13,7 @@ coroutine
 coroutines
 cron
 crons
+datetime
```

Comment:
this should probably be backticked instead

Reply:
I'm not sure where it is, and I'll search harder :)

Reply:
Remove it from the wordlist and re-run the spellcheck, the error will pinpoint the location of the faulty string :)

Reply:
Ok, I found where the problem. It's in the [Job dataclass](https://github.com/peopledoc/procrastinate/blob/97cb7547c2ba1a77f9edf6c18887855c2871e1c6/procrastinate/jobs.py#L75-L76), whose `scheduled` property is automatically documented. I see no other way to fix this than adding `datetime` to the spelling word list.